gitlab/types.py

```diff
@@ -38,7 +38,10 @@ def set_from_cli(self, cli_value):
             self._value = [item.strip() for item in cli_value.split(',')]
 
     def get_for_api(self):
-        return ",".join(self._value)
+        try:
+            return ",".join(self._value)
+        except TypeError:
+            return self._value
```

Comment:
If `self._value` is not a list nor None, then we end up returning unusable data. I'd prefer to have an exception raised is `self._value` is not a list/set.

Reply:
I agree with @gpocentek

Reply:
@gpocentek @max-wittig Thanks, guys. I agree with you, too. I should convert the labels to an empty list when it's `None`.

I close this one, and thanks for your time.